cypress/integration/vendor-cwu-CR.spec.js

```diff
@@ -0,0 +1,77 @@
+/// <reference types="cypress" />
+
+describe('As a user authenticated via GitHub',  function() {
+    beforeEach(function() {
+        cy.sqlFixture('dbReset.sql')
+        cy.login('vendor')
+        cy.sqlFixture('cwuOpportunity.sql')
+    })
+
+    it('create and read CWU proposal', function() {
+        cy.get('#user-sign-up-step-two-terms').click()
+        cy.get('a').contains('Complete Profile').click()
+        cy.get('a[href="/opportunities"]').contains('View All Opportunities').click()
+        cy.contains('Fixture CWU Opportunity Title').should('be.visible')
+        cy.contains('Fixture CWU Opportunity Title').click()
```

Comment:
Putting an assertion before an attempt to click generally solves the local vs. CI timing issues. I only put it in the places where I had problems, not everywhere in the test.

Reply:
great!